client/lib/signup/step-actions.js

```diff
@@ -179,6 +180,12 @@ module.exports = {
 			var errors = error && error.error ? [ { error: error.error, message: error.message } ] : undefined,
 				bearerToken = error && error.error ? {} : { bearer_token: response.bearer_token };
 
+			if ( ! errors ) {
+				// Fire after a new user registers.
+				analytics.tracks.recordEvent( 'calypso_user_registration_complete' );
```

Comment:
Are we able to pass here some props? Like site_URL, user ID, purchase?
It would be cool to have this data for slice&dice in analytics